tests/stubtest_whitelists/py3_common.txt

```diff
@@ -203,22 +203,11 @@ http.cookies.Morsel.setdefault
 http.cookies.Morsel.update
 http.server.HTTPServer.__init__
 imaplib.IMAP4_SSL.ssl
-importlib.abc.FileLoader.get_filename
-importlib.abc.Loader.exec_module
-importlib.abc.MetaPathFinder.find_spec
-importlib.abc.PathEntryFinder.find_spec
-importlib.machinery.BuiltinImporter.find_module
-importlib.machinery.BuiltinImporter.find_spec
-importlib.machinery.ExtensionFileLoader.__init__
-importlib.machinery.ExtensionFileLoader.get_filename
-importlib.machinery.FrozenImporter.find_module
-importlib.machinery.FrozenImporter.find_spec
-importlib.machinery.FrozenImporter.module_repr
-importlib.machinery.SourceFileLoader.set_data
-importlib.machinery.WindowsRegistryFinder.find_module
-importlib.machinery.WindowsRegistryFinder.find_spec
-importlib.util.spec_from_file_location
-importlib.util.spec_from_loader
+importlib.abc.FileLoader.get_filename  # Wrapped with _check_name decorator which changes runtime signature
```

Comment:
The stubs should account for decorators that change the signature. (At least for the `get_filename`s I've seen) we don't need to add `*args` / `**kwargs` since they're unused, but we should rename fullname to name in case people pass it by keyword.